app/templates/components/preprint-navbar.hbs

```diff
@@ -9,7 +9,7 @@
         </div>
         <div id="preprint-navbar-links" class="navbar-collapse collapse navbar-right">
             <ul class="nav navbar-nav">
-                <li>{{#link-to "submit" class="" invokeAction=(action "click" "link" "Navbar - Add preprint")}}{{t "global.add_preprint"}}{{/link-to}}</li>
+                <li><a href='/preprints/submit' class="" invokeAction={{action "click" "link" "Navbar - Add preprint"}}>{{t "global.add_preprint"}}</a></li>
```

Comment:
@binoculars and I looked at this today.  We haven't figured out a great way around it yet, so moving forward.  It technically works - but might be slower. The disadvantage to using an html 'a' tag is that everything is reloaded so you lose some of the benefits of a single page app.